### ISSUE TITLE
Combobox, Combobox Item Group and Combobox Item - Ensure active user interface components have sufficient contrast - (2036569896)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7787]
### Violation: 
 Ensure active user interface components have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/8ea3a009-91c1-437a-a437-efc374b44b46)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 6 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Combobox, Combobox Item Group and Combobox Item - Ensure active user interface components have sufficient contrast - (2036569896) 
- Module: Combobox, Combobox Item Group and Combobox Item 
**Issue**
There are user interface components with a contrast ratio below 3.00:1.

Foreground:#CACACA
Background: #FFFFFF
The contrast ratio is: 1.6:1
Examples of this user interface component include:
- The "check" mark icon in unselected state (available after activating combobox)

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 12L2 8.689l.637-.636L5.5 10.727l8.022-7.87.637.637z"></path>
</svg>
 ``` 
**Suggestion**
Ensure active user interface components have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying components include text field borders, checkmarks for checkboxes, fillings for radio buttons, focus indicators, and icon-only controls.

For borders, the "adjacent color" can be the color that touches the outside of the border or the color that touches the inside of the border. Contrast with both is not required.

Disabled controls that cannot be navigated to with the keyboard are exempt from this requirement.

Refer to the Accessible Color Picker extension or Color Contrast Checker site for assistance: https://www.accessibility.dev/

**Compliant Code Example** 
 ```
N/A
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This seems like a valid issue, and should be vetted through the design team.

**Comment by geospatialem** [comment]
This will be mitigated as part of https://github.com/Esri/calcite-design-system/issues/6245#issuecomment-2174528812, where the icon is removed entirely. Closing the above in favor of #6245.
